scipy/stats/_axis_nan_policy.py

```diff
@@ -172,6 +173,11 @@ def _axis_nan_policy_factory(result_object, default_axis=0,
         raise an error. This argument prevents the error from being raised when
         input is not 1D and instead places a NaN in the corresponding element
         of the result.
+    n_outputs : int, default: 2
+        The number of outputs produced by the function given 1d sample(s). For
+        example, hypothesis tests that return a namedtuple or result object
+        with attributes `statistic` and `pvalue` use the default n_outputs=2;
+        summary statistics with scalar output use `n_outputs=1`.
```

Comment:
I think statistic and pvalue should get a double tick even though they are from `result_object`
```suggestion
    n_outputs : int, default: 2
        The number of outputs produced by the function given 1d sample(s). For
        example, hypothesis tests that return a namedtuple or `result_object`
        with attributes ``statistic`` and ``pvalue`` use the default ``n_outputs=2``;
        summary statistics with scalar output use ``n_outputs=1``.
```

Reply:
Sure. It won't render, but can't hurt. I think "result object" should stay as-is, though, because I don't mean `result_object` the parameter, really.